Doc/library/pyexpat.rst

```diff
@@ -321,6 +321,11 @@ callback).
    Current byte index in the parser input.
 
 
+.. attribute:: xmlparser.CurrentByteCount
+
+   Current byte count of the parser event.
```

Comment:
```suggestion
    Number of bytes in the current event.
```

Should we say that it's 0 if the event is inside a reference to an internal entity? or if its the end-tag event for empty element tags?

Reply:
I think it makes sense to document the 0 behavior. I do want to swap the two options though, since I think the empty-element case is way more common than the internal entity case (who even knows what an internal entity is these days)